test/fixtures/tape-standalone.js

```diff
@@ -10,9 +10,31 @@ if (navigator.userAgent.includes("Chrome/")) {
 }
 
 // Export as a global a wrapped test function which enforces a timeout by default.
-window.test = (desc, fn) => {
-  tape(`${desc} (${browser})`, async (t) => {
-    t.timeoutAfter(DEFAULT_TIMEOUT);
+// eslint-disable-next-line
```

Comment:
Don't use a blanket eslint disabling. Always be explicit with the linting rule that you intend to turn off. I think that you wanted to disable `valid-jsdoc` here; that exception can be removed when you follow my suggestion of simplifying the parameter type.